docs/modules/ROOT/pages/shipping-logs-to-amazon-elasticsearch-service.adoc

```diff
@@ -1,71 +1,116 @@
-= Send logs to Amazon ElasticSearch Service
+= Send Logs of Dedicated Clusters to Amazon OpenSearch
+:description: Set up a remote logging solution using Amazon OpenSearch Service. In this tutorial, you'll set up an Amazon OpenSearch domain and configure your {hazelcast-cloud} cluster to automatically send log files to it. You'll also learn how to view these logs in Kibana.
+:page-dedicated: true
 :url-github-elasticsearch: https://github.com/elastic/elasticsearch
-:url-amazon-elasticsearch: https://aws.amazon.com/elasticsearch-service
+:url-amazon-opensearch: https://aws.amazon.com/opensearch-service
+:url-aws-signup: https://portal.aws.amazon.com/billing/signup 
 :url-aws-console: https://us-west-2.console.aws.amazon.com/es/home
-:page-dedicated: true
 
-Amazon provides link:{url-github-elasticsearch}[ElasticSearch] as a service which is called link:{url-amazon-elasticsearch}[ElasticSearch Service]. You can create ElasticSearch services with some built-in plugins like Kibana. In this tutorial, you will see how easy it is to set up an ElasticSearch service on AWS then ship {hazelcast-cloud} cluster logs to ElasticSearch. 
+{description}
 
-== Set Up an ElasticSearch Service Instance
+link:{url-amazon-elasticsearch}[Amazon OpenSearch Service] is based on link:{url-github-elasticsearch}[Elasticsearch], and allows you to search, analyze and visualize your cluster logs using Kibana.
 
-Go to link:{url-aws-console}[] and click *Create a new domain*
+[[set-up-opensearch-domain]]
+== Step 1. Set Up an Amazon OpenSearch Domain
 
-image:elasticsearch.png[Homepage for Amazon ElasticSearch Service]
+Create an Amazon OpenSearch domain to receive log files from your {hazelcast-cloud} cluster.
 
-On the initial page of domain creation, you can select your deployment type. For testing purpose, you can select *Development and Testing*, or if it is a live environment you can select *Production* option, *Custom* otherwise. And *7.7* is recommended for Elasticsearch version.
+. link:{url-aws-signup}[Create an AWS account] if you don't have one already.
+. Sign in as root user.
```

Comment:
Maybe it might seem trivial, but I don't think this is a good idea to encourage user to login via their root users. Instead, we can tell them to create IAM user and login with IAM credentials. You can see the details from AWS here https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html.

Reply:
Could we suggest that they use root user just for testing purposes but then create an IAM account for Production?

The reason for this, is that I started to look into the account configuration and it’s not that straightforward. It looks like you need to create an IAM account with access to the OpenSearchService through a policy, group or similar and somehow enable a service-linked role for it. I couldn’t work out how to enable the service-linked role.